js/index.js

```diff
@@ -85,9 +85,60 @@ function selectElementAndShowProperties(element) {
     });
   });
 
+  $('#properties-duplicate').off('click').click(() => {
+    duplicateElement(element);
+    prompt.updateCallback();
+  }); 
+
   $('#properties').fadeIn(SHOW_HIDE_DURATION);
 }
 
+function duplicateElement(element){
+  // Make a copy of the element 
+  const copiedElement = element.clone();
+  
+  // Remove the class element-selected
+  copiedElement.removeClass('element-selected');
```

Comment:
Do you think that would make it difficult for the user if they wanted to duplicate the current cell more than once?

Reply:
The original element and the new element added after duplication are identical. So it should not matter which of them you duplicate in case you want to have several duplicates. Regarding the UX design, I don't know which one is the better option.